en/python_introduction/README.md

```diff
@@ -252,7 +252,11 @@ This shows that you just created an empty dictionary. Hurray!
 
 Now, try writing the following command (try replacing your own information too):
 
-    >>> participant = {'name': 'Ola', 'country': 'Poland', 'favorite_numbers': [7, 42, 92]}
```

Comment:
I am not sure if this doesn't bring problems for the user. There might be several problems with indentation when you force people to enter newlines. I would leave it as it was before...